packages/frontend/src/state/reducers/modals/actions/index.ts

```diff
@@ -0,0 +1,5 @@
+import { createAction } from "@reduxjs/toolkit";
+
+export const setModalOpen = createAction<{
```

Comment:
There will always be one payload attribute and it will be the open state, so we can simplify and rewrite this as:

```
export const setModalOpen = createAction<boolean>("modals/setModalOpen");
```